composites/Plugin/Shared/components/HelpText.js

```diff
@@ -1,30 +1,42 @@
 /* External dependencies */
-import React from "react";
+import React, { PureComponent } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 
+/* Internal dependencies */
+import colors from "../../../../style-guide/colors";
+
 const YoastHelpText = styled.p`
-	font-size: 13px;
+	color: ${ props => props.color };
+	font-size: ${ props => props.fontSize };
+	margin-top: 0;
+	
+	a {
+        color: ${ colors.$palette_blue_medium };
+    }
 `;
```

Comment:
lines 15 and 16 use spaces instead of tabs

Reply:
Also, thinking better at this, not sure we should use a color in the first place. We should just let the links inherit the colors from WordPress, as we do for example for the "Go Premium" link. Applies also to the ChangeLanguageLink.

Reply:
The issue we adressed specifically requested that color blue. @hedgefield , an opinion on this?

Reply:
This point still needs an answer. @hedgefield seems to me we should let links inherit the default WordPress colors for links (default: `#0073aa`), as we're already doing for all the links in the meta box. Why we should use a different color?